From b39ab6f5909182def760c7996aad5fe2a0e4d8b9 Mon Sep 17 00:00:00 2001
From: Daniel Brackenbury <daniel.brackenbury@gmail.com>
Date: Tue, 7 Oct 2025 18:00:57 -0400
Subject: [PATCH] Add Nulea M501 trackball to hwdb

(cherry picked from commit 9dde7d9ae850efd3bc35a419ba4acb4427f1d5ad)

Resolves: RHEL-72702
---
 hwdb.d/70-mouse.hwdb | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index dc9be4f915..98c0149d90 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -821,6 +821,19 @@ mouse:usb:v22d4p1308:name:Laview Technology Mionix Avior 7000:*
 mouse:usb:v0e8fp00a7:name:DaKai 2.4G RX:*
  MOUSE_DPI=*800@126 1600@126
 
+
+##########################################
+# Nulea
+##########################################
+
+# Nulea M501 Wireless Trackball (USB Receiver)
+mouse:usb:v25a7pfa61:name:Compx 2.4G Receiver Mouse:*
+  ID_INPUT_TRACKBALL=1
+
+# Nulea M501 Wireless Trackball (Bluetooth)
+mouse:bluetooth:v000ep3412:name:Nulea BT5.0 Mouse:*
+  ID_INPUT_TRACKBALL=1
+
 ##########################################
 # Oklick
 ##########################################
